docs/robots/jackal/user_manual_jackal.mdx

```diff
@@ -1,12 +1,358 @@
 ---
 title: User Manual
 sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 4
 ---
 
 import ComponentIntroductionJackal from "/components/introduction_jackal.mdx";
+import Support from "/components/support.mdx";
+import ComponentUsingRos from "/components/using_ros.mdx";
+import ComponentCommonSafeWorkProcedures from "/components/common_safe_work_procedures.mdx";
+import ComponentSafetyWarning from "/components/safety_warning.mdx";
 
 <ComponentIntroductionJackal />
 
-:::warning todo—under construction
+## Introduction
+
+Jackal is a rugged, lightweight, fast and easy-to-use unmanned ground robot for rapid prototyping
+and research applications. Jackal includes a computer, as well as basic IMU and GPS. Standard
+perception modules are available, including URDF, simulator integration, and demonstration
+applications.
+
+<ComponentSafetyWarning />
+
+### Shipment Contents
+
+Your Jackal shipment contains the following:
+
+- Clearpath Robotics Jackal
+- 270 Wh lithium battery pack
+- 110V/220V universal charger
+- Sony PS4 Bluetooth controller
+
+If you purchased standard payload modules or custom integration services with Jackal, then additional
+equipment will be included per your specific configuration, plus further documentation as required.
+
+### Hardware Overview
+
+#### System Architecture
+
+Jackal is built around a computer running Ubuntu (Intel-based computer or Jetson developer kit), paired
+with a 32-bit MCU. The MCU handles IO, power supply monitoring and motor control, as well as supplying
```

Comment:
Suggest this be changed to:
'with a 32-bit microcontroller MCU.

Reply:
Updated for Husky, Jackal, and Dingo